src/fixtures.js

```diff
@@ -1,5 +1,5 @@
 export const MOCK_GEN_DK_RESPONSE = {
-  CiphertextBlob: Buffer.from([
+  CiphertextBlob: Buffer.from(new Uint8Array([
```

Comment:
- i am leave the types as Buffer so that test cases do not have to change
- but adding the Uint8Array to highlight the change of the sdk